src/tiledimage.js

```diff
@@ -162,6 +163,7 @@ $.TiledImage = function( options ) {
         crossOriginPolicy:      $.DEFAULT_SETTINGS.crossOriginPolicy,
         placeholderFillStyle:   $.DEFAULT_SETTINGS.placeholderFillStyle,
         opacity:                $.DEFAULT_SETTINGS.opacity,
+        preload:                $.DEFAULT_SETTINGS.preload,
```

Comment:
We're moving towards having our private properties prefixed with an underscore (to encourage people to use the accessor functions); please use `_preload` like `_fullyLoaded`.